### PR TITLE
add turkish language to audio-transformers-course

### PR DESCRIPTION
**Pull request by tanersekmen**
I have translated from chapter 0 to chapter 2 but If I have enough time to translate, I will do it. 🤗

### CODE REVIEW

**Comment by MKhalusova** [comment]
Hi @tanersekmen ! Thanks a lot for translating the first few chapters. Two more files need to be updated in order to build the course in a new language - `.github/workflows/build_documentation.yml` and `.github/workflows/build_pr_documentation.yml`.  Please add `tr` to the list of languages similar to how it's done [here](https://github.com/huggingface/audio-transformers-course/blob/360ba6cffda44d249029f790f03a93b08ec74cb8/.github/workflows/build_documentation.yml#L17).

**Comment by tanersekmen** [comment]
Hi @MKhalusova, okay. 
I missed that part.

**Review by tanersekmen — COMMENTED** [review]
I add that part. can you check the update @MKhalusova ?

File: chapters/tr/chapter1/quiz.mdx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+# Kurs materyalini anladığınızı kontrol edin
+
+### 1. Örnekleme hızı hangi birimlerle ölçülür?
+
+<Question
+	choices={[
+		{
+			text: "dB",
+			explain: "Hayır, genlik desibel (dB) cinsinden ölçülür.
```
[the remark this answers is not in the document]
Comment (MKhalusova) [reply chapters/tr/chapter1/quiz.mdx:11]: An unclosed string breakes the doc build.

**Comment by tanersekmen** [comment]
Hi, @merveenoyan, can you check my PR when you have time?  I can not see any problem in the preview section.

**Review by MKhalusova — APPROVED** [review]
The translation looks good to me, I think we can merge now.

**Comment by merveenoyan** [comment]
@tanersekmen sanırım çeviride openai gibi bir model kullanıp kendiniz çevirmemişsiniz, benim bu PR'ı kaçırmam kötü olmuş. Bunu düzeltmek ne yazık ki çok vaktimi alacak, lütfen bir daha böyle bir PR açmayın.

**Pull request merged**